src/update_sdk.cmd

```diff
@@ -0,0 +1,43 @@
+REM This script replaces the SDK folder with the latest bits from the Azure SDK repository.
+REM It removes some files we do not need.  It currently targets the develop branch until fixes are
+REM merged into master.
+
```

Comment:
Sorry for the delay on this - we are working on making some changes to the SDK so Azure maintains an arduino version that can be used directly as a submodule.  We'll have a PR for that in a few weeks, it just isn't required for launching this week so will come next.